URBNSwiftyConvenience/Classes/URBNBorders.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  URBNBorders.swift
+//  Pods
+//
+//  Created by Lloyd Sykes on 6/30/17.
+//
+//
+
+import Foundation
+
+extension UIView {
+    public var urbn_leftBorder: URBNBorder? {
```

Comment:
Larger group question: Do we want to keep this working exactly how it works in URBNConvenience? Or make it more swifty (Get rid of `urbn_` prefix, etc…)